core/tests/test_utils.py

```diff
@@ -2265,6 +2265,9 @@ def post_json(
 
         return self._parse_json_response(json_response, expect_errors)
 
+    # This function is requried to test the webhook for the bulk email provider,
```

Comment:
Ah, now I see what's happening. Can you get self.post_json() to call this function internally, then?

Reply:
Done, removed this function and added an argument to `post_json`.